fern/pages/docs/getting-started/customer-showcase.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Customer Showcase
+---
+
+At Fern, we help businesses build and distribute world-class documentation that developers love. From startups to global enterprises, our customers trust Fern to elevate their developer experience and _decrease time to_ `200`. Below are some of the companies leveraging Fern to power their documentation.
```

Comment:
About Fern Docs

Fern Docs is a documentation tool for creating beautiful, customizable developer docs. Style your docs to match your brand, write content in MDX, add your OpenAPI spec, and integrate seamlessly with CI/CD. Below are some companies building with Fern to elevate their developer experience.

Reply:
i pushed an update that aligns more with that framing